### ISSUE TITLE
Confused about the Observation Encoder

### ISSUE THREAD

**Issue by wzn0828** [wayveai/mile#33]
Hi，many thanks for your excellent work, Mile.

However, I'm very confused about the Observation Encoder:

In your paper, you described that the observation embedding x_t is the concatenation of the image feature (after pooling to BEV), route map feature and speed feature: x_t = [x_t' , r_t, m_t].
![image](https://github.com/wayveai/mile/assets/29533484/1b0b6f11-4401-46e8-8675-7dcfd5d2c058)

That's to say, the order is : pooling to BEV → mapping to a 1D vector → concat route map feature and speed feature

But, in the code, it seems that the order is reversed: concat route map feature and speed feature → pooling to BEV → mapping to a 1D vector. Like the codes in Mile.mile.models.mile.py:
![image](https://github.com/wayveai/mile/assets/29533484/7a1200db-9f63-4ac9-b704-4e994e058343)

Very confused, or am I wrong? pls tell me...

**Comment by anthonyhu** [comment]
Good catch, I think in practice this is a technicality that does not matter much (concatenating before of after the BEV backbone).
